values.yaml

```diff
@@ -766,46 +766,53 @@ meshGateway:
   # Number of replicas for the Deployment.
   replicas: 2
 
-  # What gets registered as wan address for the gateway.
+  # What gets registered as WAN address for the gateway.
   wanAddress:
-    # Port that gets registered.
+    # source configures where to retrieve the WAN address (and possibly port)
+    # for the mesh gateway from.
+    # Can be set to either: Service, NodeIP, NodeName or Static.
+    #
+    # Service - Determine the address based on the service type.
+    #   If service.type=LoadBalancer use the external IP or hostname of
+    #   the service. Use the port set by service.port.
+    #   If service.type=NodePort use the Node IP. The port will be set to
+    #   service.nodePort so this cannot be null.
```

Comment:
What is "this" referring to here that cannot be null?

Reply:
`service.nodePort`. Will make clearer.